### ISSUE TITLE
Add zpool state for zfs plugin on Linux

### ISSUE THREAD

**Issue by cliffjao** [influxdata/telegraf#10619]
## Feature Request

The state of the zpool is available in /proc/spl/kstat/zfs/rpool/state. It would be great to read this state and send it over to InfluxDB.

Example:
```
# cat /proc/spl/kstat/zfs/rpool/state
ONLINE
```

### Use case:

Would like to set alerts in InfluxDB when a zpool gets degraded

**Comment by sspaink** [comment]
@cliffjao this seems like it would be a good feature, would you be interested in implementing this change perhaps?

**Comment by sspaink** [comment]
@cliffjao are you still interested in this feature?

**Comment by cliffjao** [comment]
> @cliffjao are you still interested in this feature?

Yes, this feature would be great but I don't have the bandwidth to get a dev environment set up for Telegraf and implement it. I hacked together a shell script to read the state and convert to JSON output for now.
